source/dub/packagesuppliers/isupplier.d

```diff
@@ -0,0 +1,54 @@
+module dub.packagesuppliers.isupplier;
+
+public import dub.dependency : Dependency, Version;
+public import dub.internal.vibecompat.core.file : NativePath;
```

Comment:
Should be `dub.internal.vibecompat.core.inet.path : NativePath, toNativeString;`. Should be noted that this is the first use of public imports (I used to public import all types necessary for the public API in a module, but stopped doing over time so for no real reason). The `toNativeString` might also be cleaner to import directly in `filesystemsupplier`.